paasta_tools/utils.py

```diff
@@ -201,6 +210,13 @@ def check_mem(self):
                 return False, 'The specified mem value "%s" is not a valid float.' % mem
         return True, ''
 
+    def check_disk(self):
+        disk = self.get_disk()
+        if disk is not None:
+            if not isinstance(disk, float) and not isinstance(disk, int):
+                return False, 'The specified disk value "%s" is not a valid float.' % disk
```

Comment:
The conditional allows for floats and ints. It is only the error message that is specific to floats. This code is based on the check above for memory. I can fix up both of the messages to be a bit more clear.